### DISCUSSION TITLE
Can you display the git repo name instead of branch?

### DISCUSSION THREAD

**Discussion by snuggie12** [JanDeDobbeleer/oh-my-posh#1186] [Troubleshoot]
Doesn't seem like it's possible in the regular config and I'm unsure how the templates work. Curious if this is possible or if it should be a feature request.

**Comment by JanDeDobbeleer** [comment]
Isn't the repo name the folder name? Or what are you after?

**Reply by SweetestSufferance** [reply]
I'm guessing @snuggie12  is asking so that when you're using folder icons to replace folders in the path, you can see what repository you're in.  I was just looking for the same thing, but might just build custom.

**Reply by JanDeDobbeleer** [reply]
@SweetestSufferance si also when in sub folders I suppose? I'll add that.

**Reply by JanDeDobbeleer** [reply]
@SweetestSufferance @snuggie12 https://github.com/JanDeDobbeleer/oh-my-posh/pull/2462
